src/cli-constant.js

```diff
@@ -186,15 +186,24 @@ const detailedOptions = normalizeDetailedOptions({
     forwardToApi: true,
     exception: value => typeof value === "string", // Allow path to a parser module.
     choices: [
-      "flow",
-      "babylon",
-      "typescript",
-      "css",
+      { value: "flow", description: "https://github.com/facebook/flow" },
+      { value: "babylon", description: "https://github.com/babel/babylon" },
+      {
+        value: "typescript",
+        description: "https://github.com/eslint/typescript-eslint-parser"
+      },
+      { value: "css", description: "https://github.com/postcss/postcss" },
```

Comment:
This description is incorrect. We don’t use the postcss parser (and never have). The "css" option tries to use postcss-less first, and if that fails, postcss-scss (unless a regex says to flip the order).

Reply:
Oops, thanks for catching that.